model/delivery/LtiTestTakerAuthorizationService.php

```diff
@@ -44,7 +41,6 @@
 class LtiTestTakerAuthorizationService extends TestTakerAuthorizationService implements DelegatedServiceHandler
 {
     public const CUSTOM_LTI_PROCTORED = 'custom_proctored';
-    public const CUSTOM_LTI_AUTOSTART = 'custom_autostart';
```

Comment:
Removing a public constant is technically a breaking change. I don’t see any external usage of that constant, though, so we should be good.